src/shared/components/case-editor/services/cases.service.ts

```diff
@@ -217,15 +218,88 @@ export class CasesService {
     if (!eventTrigger.wizard_pages) {
       eventTrigger.wizard_pages = [];
     }
-    /* FIXME: find a better place for this code */
```

Comment:
if you want to remove this please create a tech debt ticket so we don't lose track of it

Reply:
I have found a better place. In its own method :) It is not the code we want to reuse, so I think it is enough to split functionality into separate methods. I personally don't like a vague comments like that. It doesn't specify what the solution should be.

Reply:
ok cool